php/src/Docker/DockerActionManager.php

```diff
@@ -62,17 +54,21 @@ public function GetContainerRunningState(Container $container) : IContainerState
             $response = $this->guzzleClient->get($url);
         } catch (RequestException $e) {
             if ($e->getCode() === 404) {
-                return new ImageDoesNotExistState();
+                return IContainerState::ImageDoesNotExist;
             }
             throw $e;
         }
 
         $responseBody = json_decode((string)$response->getBody(), true);
 
-        if ($responseBody['State']['Running'] === true) {
-            return new RunningState();
-        } else {
-            return new StoppedState();
+        if ($responseBody['State']['Running'] !== true) {
+            return IContainerState::Stopped;
+        }
+        else if(array_key_exists('Health', $responseBody['State']) && $responseBody['State']['Health']['Status'] !== 'healthy') {
```

Comment:
```suggestion
        } elseif (array_key_exists('Health', $responseBody['State']) && $responseBody['State']['Health']['Status'] !== 'healthy') {
```

In php `else if` is not the same as `elseif`.